### PR TITLE
Fix winrate computation typo, Change blacklist to use relative difference instead of absolute differences

### PR DESCRIPTION
**Pull request by oswinso**
Using absolute differences for the backlist mechanism results in the sensitivity of blacklisting to depend on the current magnitude of the scores.

For example, if the current score is 0.2, asking for a 0.1 **absolute** difference in the scores amounts to asking for a **50%** increase / decrease in score, which is extremely difficult. On the other hand, if the current score is 0.9, then this becomes a milder requirement of 11% increase / decrease in score.

When the validation criteria updated resulting in different magnitudes of scores for the current models, the resulting threshold for blacklisting will then also need to be updated if we are using **absolute** differences.

Instead, this PR changes the blacklisting mechanism to use **relative** differences instead, which can be seen as automatically choosing the threshold to scale with the current "difficulty".

Also, this fixes a typo with the winrate computation that results in the winrate being greater than 1

### CODE REVIEW

**Comment by 0X0StradSong** [comment]
Agree with the `win_rate` one.

For the clone checking, it checks the maximum diff of two tensors. For example, if the first model's scores are `[0.1, 0.3, 0.8]` and second model's scores are `[0.1, 0.3, 0.6]`, current detector says the two models are different. This is a very mild threshold. Even a model training for a new epoch, the new model will generate very different scores compared to the previous one. Correct me if I was wrong :)

**Comment by oswinso** [comment]
>  For example, if the first model's scores are [0.1, 0.3, 0.8] and second model's scores are [0.1, 0.3, 0.6], current detector says the two models are different.

This is true. However, in your example, you have the scores (i.e., the product of the `pann_mmd`, `word_error_rate`, `tone_color` and `antispoofing` compnents) go as high as `0.8`. This was true before the introduction of the new anti-spoofing component. However, currently, the scores only go up to around `0.25` due to this new anti-spoofing component. If I multiply your scores by `0.25`, as in the case where we introduced the new antispoofing component to the scores, I get

`[0.025, 0.075, 0.2]` vs `[0.025, 0.075, 0.15]`

In other words, **new components to the score will change the result of model duplication**. In this case, the addition of this new component says that the two models are the same, even though they were different before adding this new component.

On the other hand, by switching to a relative threshold, **multiplication by a constant does not affect the results**. This makes it easier to add new components to the score without needing to also modify the threshold every time.
